Library/Homebrew/config.rb

```diff
@@ -50,7 +50,11 @@ def mkpath
 HOMEBREW_TEMP = Pathname.new(ENV.fetch("HOMEBREW_TEMP", "/tmp"))
 
 unless defined? HOMEBREW_LIBRARY_PATH
-  HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent.join("Homebrew")
+  HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
+end
+
+unless defined? HOMEBREW_CABAL_CONFIG_PATH
```

Comment:
IMO this is better placed in `utils` if we include it. `config` is almost the exclusive province of functions and elements that influence how `brew` _itself_ behaves.